sdks/python/apache_beam/transforms/dataflow_distribution_counter_test.py

```diff
@@ -39,26 +42,23 @@ def test_calculate_bucket_index_within_max_long(self):
       power_of_ten *= 10
 
   def test_add_input(self):
-    counter = DistributionAccumulator()
+    counter = DataflowDistributionCounter()
     expected_buckets = [1, 3, 0, 0, 0, 0, 0, 0, 1, 1]
     expected_sum = 1510
     expected_first_bucket_index = 1
     expected_count = 6
     expected_min = 1
     expected_max = 1000
-    for value in [1, 500, 2, 3, 1000, 4]:
-      counter.add_input(value)
-    self.assertEquals(counter.buckets, expected_buckets)
+    for element in [1, 500, 2, 3, 1000, 4]:
```

Comment:
Could you add a test for the edges? (Largest possible input etc.)

Reply:
There is a test case https://github.com/apache/beam/pull/5028/files#diff-9e972e8dfd5893228ef8262c3f1c30fcR31 which tests calculating bucket index of max_input. I think the critical part of DistributionCounter is calculating bucket index, so will this be enough for covering edge cases?